app/app.cpp

```diff
@@ -1,11 +1,36 @@
 #include "app.h"
 #include <format>
+#include <sstream>
 
 namespace vsite::oop::v1
 {
 /*
 * function implementations
 */
+	std::string to_hex(int num) {
+		return std::format("{:X}", num);
+	}
 
+	std::string to_exp(double num){
+		return std::format("{:.2e}", num);
+	}
+
+    void mult_table(int n, std::ostream& ss) {
+        if (n < 0 || n > 20 || n == 0) {
+            return;
+        }
+        ss << std::format("{}", 0);
```

Comment:
`for (int j = 0; j <= n; ++j)`
pa ne treba ovaj ispis (i formatiranje) konstante